.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -1,37 +1,49 @@
 ---
 name: Device Request
-about: This form helps in requesting a new device to be supported.
-title: "[DEVICE SUPPORT REQUEST] - Devicename"
+about: Request a new device to be supported.
+title: "Device Support: Device name"
 labels: Device Request
 assignees: ''
 
 ---
 
-Before requesting a device, please make sure to search the open and closed issues for any requests in the past. Sometimes devices have been requested before but are not implemented yet due to various reasons. If there are no hits on your Product Number, please proceed.  
+<!--
+  - Before requesting a device, please make sure to search the open and closed issues for any requests in the past.
+  - Sometimes devices have been requested before but are not implemented yet due to various reasons.
+  - If there are no hits on your for the device, please proceed.
+  - If you're unsure if the device support is already requested, please ask for advise in our Discord chat: https://discord.gg/QFhTxqN
 
-**Device Name:**
+-->
 
-**Device Type:**
-- [ ] Light
-- [ ] Lock
-- [ ] Remote
-- [ ] Sensor
-- [ ] Siren
-- [ ] Thermostat
-- [ ] Other: 
+## Device
 
-**Device Manufacturer as per deCONZ GUI basic cluster:**
+- Product name: The device name as shown on the product or package.
+- Device Type: (Light / Lock / Remote / Sensor / Siren / Thermostat)
+- Manufacturer: As per deCONZ GUI Basic cluster.
+- Model identifier: As per deCONZ GUI Basic cluster.
 
-**Device model identifier as per deCONZ GUI basic cluster:**
+<!--
+  Please refer to https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Request-Device-Support
+  on how the Basic Cluster attributes are optained.
+-->
 
-**If you do not know what is needed, please read [this](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Request-Device-Support)**
+## Screenshots
 
-**Screenshot of Clusters of the Node:**
+<!--
```

Comment:
The old one as i mentioned helps in creating a path for the dev to implement. Now people might place the screens in different order everytime.

Reply:
The order of screens haven't been a issue in the past (imho) I'd like to keep this dense. The screenshots are already very self describing. Giving them headlines isn't needed.

Reply:
I dont use them so it is not me to judge. Need @SwoopX here.

Reply:
To be honest, order doesn't matter for me but information. As long as everything's there, I'm cool.

Reply:
Instead of screenshots  could we get a "dump" function that generates a text file that can be provided?